CatAnalyzer/prod/job_fcnc.sh

```diff
@@ -0,0 +1,87 @@
+./create-batch --jobName SingleMuon_Run2017B  --fileList $CMSSW_BASE/src/CATTools/CatAnalyzer/data/dataset/dataset_SingleMuon_Run2017B.txt  --cfg fcncAnalyzer_cfg.py --transferDest /store/user/minerva1993/ntuple_jw/2017/V9_2/production/SingleMuon_Run2017B --maxFiles 30 --args 'UserJSON=true' 'runOnTTbarMC=0' 'TTbarCatMC=0'
```

Comment:
do we have create-batch in the prod directory?

Reply:
Hmm, this is temporary solution to deal with some missing lines from V9_1 recipe (about submodule). We can add those lines in wiki later